files/rsyslog/rsyslog.sh

```diff
@@ -174,6 +174,9 @@ else
 fi
 RSYSLOG_VERSION=`/usr/sbin/rsyslogd -v | awk -F'[ ,]+' '/^rsyslogd / {print $2}'`
 PIPELINE_VERSION="${RSYSLOG_VERSION} ${DATA_VERSION}"
+
+IPADDR6="" # So as to omit this field from logs. See this PR for more info.
```

Comment:
Not sure how to add label, but this is still WIP. I did not know to which PR to refer.

Reply:
I added the do-not-merge/work-in-progress label to the PR